dev-environment.yml

```diff
@@ -34,4 +36,4 @@ dependencies:
   - richdem
 
   - pip:
-    - git+https://github.com/GlacioHack/geoutils.git
+    - geoutils==0.0.9
```

Comment:
Is the plan then to always link a version of xdem to a version of geoutils?

Reply:
Yes, for stable versions at least.